llama_index/evaluation/benchmarks/beir.py

```diff
@@ -82,6 +84,10 @@ def run(
             results = {}
             for key, query in tqdm.tqdm(queries.items()):
                 nodes_with_score = retriever.retrieve(query)
+                for node_postprocessor in node_postprocessors:
```

Comment:
should probably have `node_postprocessors = node_postprocessors or []` at some point before this to pass linting